test/jdk/java/lang/module/ModuleDescriptorHashCodeTest.java

```diff
@@ -63,6 +62,45 @@ public void testBootModuleDescriptor() throws Exception {
         }
     }
 
+    /**
+     * Verifies that two "equal" module descriptors which only differ in the order of
+     * {@link ModuleDescriptor.Opens.Modifier opens modifiers}, that were used to construct the
+     * descriptors, have the same hashcode.
+     */
+    @Test
+    public void testModifiersOrdering() throws Exception {
```

Comment:
Are you planning to expand the test to cover Requires.Modifier and Exports.Modifier too?